packages/client/src/components/NewArrivalsCarousel/useNewArrivals.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { apiURL } from '../../apiURL';
+
+export const useNewArrivals = () => {
+  const [movies, setMovies] = React.useState([]);
+  const [error, setError] = React.useState(null);
```

Comment:
in general, this is correct :)

In fact, you are not using this anyway in the actual component :) So it is kind of developing for the future at this point, unless you do include some sort of message for the user in the component itself when new arrivals cannot be found. I refer here to line 34 in `NewArrivalsCarousel`

Reply:
hey there :)) i guess i did this time :))

Reply:
i added user action error handling :)))